test/acceptance/features/bindAppToService.feature

```diff
@@ -661,8 +666,9 @@ Feature: Bind an application to a service
             apiVersion: "stable.example.com/v1"
             kind: Backend
             metadata:
-                name: demo-backserv-cr-2
+                name: demo-backserv-cr-3
```

Comment:
why this change?

Reply:
The `demo-backserv-cr-2` resource name is already used. And it was making the test more fragile. A unique name is always better.